extensions/amp-story/1.0/toast.js

```diff
@@ -36,8 +36,9 @@ export class Toast {
   /**
    * @param {!Window} win
    * @param {!Node|string} childNodeOrText
+   * @param {Element} storyEl
    */
-  static show(win, childNodeOrText) {
+  static show(win, childNodeOrText, storyEl) {
```

Comment:
You can actually get the window from an element, by doing `const win = toWin(rootElement.ownerDocument.defaultView)`.  Then you won't need to take `win` as a param at all.

As a nit, for some reason I also prefer the element being before the string.